pkg/importer/dataStream_test.go

```diff
@@ -25,6 +46,78 @@ func createDataStream(ep, accKey, secKey string) *dataStream {
 	return ds
 }
 
+func createDataStreamMulti(testfile, ep string) *dataStream {
+	// set our directory for images
+	imageDir, _ := filepath.Abs("../../test/images")
+	urlPath := filepath.Join(ep, testfile)
+	testFilePath := filepath.Join(imageDir, testfile)
+
+	dsurl, _ := ParseEndpoint(urlPath)
+
+	ds := &dataStream{
+		Url:         dsurl,
+		buf:         make([]byte, image.MaxExpectedHdrSize),
+		accessKeyId: "",
+		secretKey:   "",
+	}
+
+	ds.constructReaders()
+
+	buf := bytes.NewBuffer(nil)
+	f, _ := os.Open(testFilePath)
+	io.Copy(buf, f)
+	f.Close()
+	testBytes, _ := ioutil.ReadAll(buf)
```

Comment:
Copy has copied file content into buf. Why do you need to move them to testBytes?